### PR TITLE
add arm64 prebuilt workers for linux

### PR DESCRIPTION
**Pull request by arcinston**
Updates GH action to also build pre-built workers for linux instances running on arm64/aarch64

### CODE REVIEW

**Comment by ibc** [comment]
Before accepting this PR we need to fix this recent issue that didn't happen a few weeks ago:

https://github.com/versatica/mediasoup/actions/runs/8520123345/job/23337119268?pr=1363

**Comment by arcinston** [comment]
even i was looking at this issue right now @ibc

**Comment by ibc** [comment]
> even i was looking at this issue right now @ibc

I've fixed it and merged it right now.

**Comment by ibc** [comment]
Please update your branch with v3 branch. CI should pass.

However, if we are building worker prebuilt images for Linux arm64, then we must also test Linux arm64 in other CI tasks, including worker CI, Rust CI, Node CI, fuzzer CI, etc.

**Comment by arcinston** [comment]
got it updating

and looking into testing in other ci tasks @ibc

**Comment by ibc** [comment]
Prettier errors (https://github.com/versatica/mediasoup/actions/runs/8521809219/job/23340709865?pr=1363).

Please run `npm run format:node`

File: .github/workflows/mediasoup-rust.yaml [hunk 1]
```diff
@@ -37,7 +41,7 @@ jobs:
           path: |
             ~/.cargo/registry
             ~/.cargo/git
-          key: ${{ matrix.os }}-cargo-${{ hashFiles('**/Cargo.toml') }}
+          key: ${{ matrix.ci.os }}-cargo-${{ hashFiles('**/Cargo.toml') }}
```
Comment (ibc) [inline .github/workflows/mediasoup-rust.yaml:44]: 👍

**Comment by ibc** [comment]
Thanks a lot, merging.

**Comment by ibc** [comment]
@arcinston, I don't see that arm64 prebuilts are being generated, see assets in any recent version:

https://github.com/versatica/mediasoup/releases

It's even worse! The arm64 prebuilt images are replacing the x64 ones because they are generating the same name!!

CleanShot-2024-04-09-at-19 06 05@2x

I've created an issue https://github.com/versatica/mediasoup/issues/1372

**Comment by arcinston** [comment]
I realised this too the reason is this

https://github.blog/changelog/2023-10-30-accelerate-your-ci-cd-with-arm-based-hosted-runners-in-github-actions/

GitHub will launch its arm64 runners soon 
but it's not out yet it's in private beta 😞 

I did not know of this

I am so sorry

**Comment by arcinston** [comment]
for now it should be fine
as the arm64 runner is actually Linux x64

so it should not cause trouble 

but as soon as the arm64 runners are out in public 

it should work as expected

**Comment by arcinston** [comment]
cc @ibc

**Comment by ibc** [comment]
Thanks. Let's please handle the discussion in https://github.com/versatica/mediasoup/issues/1372.

**Pull request merged**